buildspec.yml

```diff
@@ -0,0 +1,17 @@
+version: 0.1
+
+phases:
+  build:
+    commands:
+      - echo Build started on `date`
+      - mvn install -DskipTests=true -B -pl $MODULE -am -Dmaven.compiler.showWarnings=false -Dmaven.javadoc.skip=true
+      - mvn verify -pl $MODULE $ARGS
+      - mvn surefire-report:report -pl $MODULE
+  post_build:
+    commands:
+      - echo Build completed on `date`
+      - ln -s ./$MODULE ./module 
+artifacts:
+  base-directory: module
+  files:
+    - target/**/*
```

Comment:
The standard location is at the [root of the repository](http://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html#build-spec-ref-name-storage). We could add a step in the manual workflow to copy the file to the repo root, zip up the repo, copy it to S3, and use that as the source for Code Pipelines. Sounds like a lot of mechanism. Can we revisit this later, perhaps adding a TODO?